src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -489,6 +489,28 @@ void MavlinkReceiver::handle_message_command_both(mavlink_message_t *msg, const
 			send_storage_information(cmd_mavlink.param1 + 0.5f);
 		}
 
+	} else if (cmd_mavlink.command == MAV_CMD_SET_CAMERA_ZOOM) {
+		struct actuator_controls_s actuator_controls = {};
+		actuator_controls.timestamp = hrt_absolute_time();
+
+		for (size_t i = 0; i < 8; i++) {
+			actuator_controls.control[i] = NAN;
+		}
+
+		if ((int)(cmd_mavlink.param1 + 0.5f) == 2) {
+			actuator_controls.control[4] = cmd_mavlink.param2 / 50.0f - 1.0f;
```

Comment:
Is it documented somewhere that channel 5 is zoom?

Reply:
No, channel 5 is still reserved.

Reply:
Hm, could you add it to the description of the channel then in the message?

Reply:
I added a PR to Devguide. Is that what you expected?

Reply:
And I would add a note or index in the msg file:
https://github.com/PX4/Firmware/blob/master/msg/actuator_controls.msg